README.rst

```diff
@@ -0,0 +1,183 @@
+
+* 23rd March 2020, we intend to unfreeze devel and merge temp-2.10-devel back into devel. From that point on, devel for ansible/ansible will be for the ansible-base project only.
+
+FAQ
+====
+
+Users of Ansible
+-----------------
+
+`Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+
+Q: Once the next version of Ansible is released, will my playbooks still work
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For users of the community version of ansible pip/apt-get install ansible will continue to give you a working install of Ansible including the three thousand plus modules.
+
+Q: I deploy Ansible directly from devel. Is that still advised?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We recognize that many users install Ansible directly from devel, and generally we do our best to keep the devel branch relatively stable. These changes, however, will have a very large short-term impact, and we cannot guarantee that devel will be as stable as it has been in the past through this migration process.  
+
+Q: When will the next version of Ansible be released with these changes?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We don't have a firm date yet, but we plan to release Ansible 2.10 sometime in 2020, and we do expect to have several alpha/beta releases between now and then. Until that time, Ansible 2.9 will continue to be the supported version.
+
+Q: I'd like to consume the development stream of Ansible during this process. How do I do that?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the split has been successfully accomplished, devel users will consume ansible-base and collections separately. We will provide instructions for installing the various components from devel when it is unfrozen on March 23.
+
+Contributors to Ansible
+------------------------
+
+`Using Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
+
+Q: I'd like to submit a new module or plugin to Ansible. How shall I proceed?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you're a vendor/partner and you're writing Ansible content to interact with your software, we recommend writing your own collection. This will allow you to pursue certification against the Ansible Automation Platform. For more info on certification, read here [FIXME: link].
+
+If you want to submit your module to the General Community Collection, you can find those guidelines here [FIXME: link].
+
+If you want to submit your module to an existing collection, you'll want to coordinate with the maintainers of those collections and follow their guidelines. Note that not all collections will necessarily accept new modules. 
```

Comment:
> Note that not all collections will necessarily accept new modules.

I don't think this sentence is necessary; it's not like it's an expectation that anything will be accepted into any open source project anyways (it wasn't that way with ansible/ansible before, either).

Reply:
I think it's important to point out that not all collections will share the same guidelines.